tests/test_inference_client.py

```diff
@@ -219,13 +219,27 @@ def test_text_to_speech(self) -> None:
         audio = self.client.text_to_speech("Hello world")
         self.assertIsInstance(audio, bytes)
 
+    def test_token_classification(self) -> None:
+        model = "dbmdz/bert-large-cased-finetuned-conll03-english"
+        output = self.client.token_classification(
+            "My name is Sarah Jessica Parker but you can call me Jessica", model=model
+        )
+        self.assertIsInstance(output, list)
+        self.assertGreater(len(output), 0)
+        for item in output:
+            self.assertIsInstance(item["entity_group"], str)
+            self.assertIsInstance(item["score"], float)
+            self.assertIsInstance(item["word"], str)
+            self.assertIsInstance(item["start"], int)
+            self.assertIsInstance(item["end"], int)
+
     def test_zero_shot_image_classification(self) -> None:
         output = self.client.zero_shot_image_classification(self.image_file, ["tree", "woman", "cat"])
         self.assertIsInstance(output, list)
         self.assertGreater(len(output), 0)
         for item in output:
-            self.assertIsInstance(item["label"], str)
-            self.assertIsInstance(item["score"], float)
+            self.assertIsInstance(item[0]["label"], str)
+            self.assertIsInstance(item[0]["score"], float)
```

Comment:
```suggestion
            self.assertIsInstance(item["label"], str)
            self.assertIsInstance(item["score"], float)
```

Not relevant to this PR I believe?